.github/dependabot.yml

```diff
@@ -5,28 +5,28 @@
 
 version: 2
 updates:
-  - package-ecosystem: "npm" # See documentation for possible values
```

Comment:
Let's stick with `"` in yaml files. They have different meanings.

Reply:
Added an `overrides` section in the prettier config for Yaml files, and re-ran